tests/functional_tests/uri.py

```diff
@@ -38,12 +39,18 @@
   from urllib.parse import quote as urllib_quote
 
 from framework.wallet import Wallet
+import pytest
```

Comment:
In monero/tests/ there is a README.md file that lists the dependencies one needs to install to run the tests. I would add pytest there.